gluten-core/src/main/scala/org/apache/spark/sql/hive/HiveTableScanExecTransformer.scala

```diff
@@ -70,11 +70,11 @@ class HiveTableScanExecTransformer(
 
   override def getPartitions: Seq[InputPartition] = partitions
 
-  override def getPartitionSchemas: StructType = relation.tableMeta.partitionSchema
```

Comment:
@yma11 I think the names of these interfaces need to be changed. Here, you can refer to the corresponding interfaces in Spark, and we should not use plurals.